frontend/src/components/NavigationBar.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="w-full h-20 bg-gray-300 flex flex-row items-center gap-5">
+    <img src="../assets/Logo.png" class="h-full mr-5" />
```

Comment:
I think we should use a different version of the Logo without the background, I will provide you with it

Reply:
I uploaded a glow and a not glowng version. I have not yet implemented that yet. we should do that based on the final design and see what looks better